qiskit_nature/algorithms/excited_states_solvers/excited_states_eigensolver.py

```diff
@@ -57,34 +57,15 @@ def solver(self, solver: Union[Eigensolver, EigensolverFactory]) -> None:
         """Sets the minimum eigensolver or factory."""
         self._solver = solver
 
-    def solve(
+    def get_qubit_operators(
```

Comment:
I have not yet found a way to refactor `ExcitedStatesSolver.get_qubit_operators` to call `self._gss.get_qubit_operators` because in principle, the ExcitedStatesSolver is not always aware of a GroundStateSolver. One way would be to split the get_qubit_operators() logic in two pieces (one getting and transforming the operators, and one instantiating the solver from the factory if needed).